test/integration/sequelize/query.test.js

```diff
@@ -409,6 +409,16 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
             .and.not.be.an.instanceOf(DatabaseError);
         });
 
+        it('get error message', async function () {
+          const sql = 'SELECT 1 FROM NotFoundTable';
+
+          try {
+            await this.sequelize.query(sql);
+          } catch (error) {
+            expect(error.message).to.not.be.undefined;
```

Comment:
this doesn't really test anything useful. No error messages is ever undefined, I don't think this is worth testing. Testing the actual stack would be much more interesting but not really possible due to it being different on all systems